test-resources/src/functions/dequeueCredentialResult/dequeueCredentialResultHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { Context, SQSEvent } from "aws-lambda";
+import { logger } from "../common/logging/logger";
+
+export const lambdaHandlerConstructor = async (
+  _dependencies: IDequeueCredentialResultDependencies,
+  _event: SQSEvent,
+  context: Context,
+): Promise<void> => {
+  logger.addContext(context);
+  logger.info("STARTED");
```

Comment:
```suggestion
  logger.info(LogMessage.DEQUEUE_CREDENTIAL_RESULT_STARTED);
```

And then the logs would need to be registered there

Reply:
Updated